library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -129,7 +129,8 @@ data class Conversations(
 
     // Sync all new and existing conversations data from the network
     suspend fun syncAllConversations(): UInt {
-        return ffiConversations.syncAllConversations()
+        // TODO: add consentState here
+        return ffiConversations.syncAllConversations(null)
```

Comment:
@nplasterer per conversation we'll add this in a follow up. or let me know if I misunderstood